sources/api/apiserver/src/server/controller.rs

```diff
@@ -579,7 +580,10 @@ mod test {
 
         // Retrieve with helper
         let settings = get_settings(&ds, &Committed::Live).unwrap();
-        assert_eq!(settings.motd, Some("json string".try_into().unwrap()));
+        assert_eq!(
+            settings.motd,
+            Some(("json string").parse::<String>().unwrap())
+        );
```

Comment:
Is the turbofish `::<String>` necessary here? I'm curious why the old code was able to deduce the type, but the `parse` invocation may be struggling to.

In general, we tend to let the type inference do its job where it can, and specify types otherwise.

Reply:
I knew there was a reason I kept the type hints/turbofishes everywhere: consistency.

Because there is one file where the types can/do not get automatically inferred:
`sources/models/src/de.rs`

attempting to use `.parse()` in e.g. `node_taints_list_representation()` results in:

```
type annotations needed
required because of the requirements on the impl of `std::cmp::PartialEq<std::vec::Vec<_>>` for `std::vec::Vec<modeled_types::kubernetes::KubernetesTaintValue>`rustc[E0283](https://doc.rust-lang.org/error-index.html#E0283)
mod.rs(40, 32): Actual error occurred here
mod.rs(225, 9): multiple `impl`s satisfying `modeled_types::kubernetes::KubernetesTaintValue: std::cmp::PartialEq<_>` found
de.rs(80, 43): consider specifying the type argument in the method call: `::<F>`
type annotations needed
cannot infer type for type parameter `T`rustc[E0282](https://doc.rust-lang.org/error-index.html#E0282)
macros.rs(50, 36): Actual error occurred here
de.rs(80, 43): consider specifying the type argument in the method call: `::<F>`
```

Reply:
What about the other locations? Like Sean said, I think we should let type inference do its job if possible.

Reply:
I'm fixing the other locations, no worries.

Would it be best to leave the ambiguous type inference I mentioned in my previous comment as a turbofish, and let all the other locations use `.parse()` without turbofishing?

Reply:
Yes, I'm ok with that.